README.md

```diff
@@ -0,0 +1,10 @@
+## Dev Setup
```

Comment:
`README.md` files are a standard practice in GitHub. Once merged, the markdown in this file will be rendered onto the project's main GitHub page. This is a code place to put developer information (like Dev setup instructions)